tests/bootstrap.php

```diff
@@ -24,6 +24,10 @@
 // Give access to tests_add_filter() function.
 require_once "{$_tests_dir}/includes/functions.php";
 
+// Start the auth mock server.
+$mockServer = shell_exec('npx oauth2-mock-server --jwk "mockserver/mocked-keys.json" -p 8887 2>&1 | tee -a /tmp/mylog 2>/dev/null >/dev/null &');
```

Comment:
If we use the approach of starting a mock oauth server for the tests, please include a test that verifies that this is actually working.